packages/main/bundle.esm.js

```diff
@@ -47,6 +47,8 @@ import Button from "./dist/Button.js";
 import Card from "./dist/Card.js";
 import Carousel from "./dist/Carousel.js";
 import CheckBox from "./dist/CheckBox.js";
+import ColorPalette from "./dist/ColorPalette.js";
+import ColorPaletteItem from "./dist/ColorPaletteItem.js";
```

Comment:
not necessary any more, but it's ok to stay. Please update the `.md` file with all components: `docs/Public Module Imports.md`